labels/de-DE.json

```diff
@@ -1,421 +1,431 @@
-﻿{
```

Comment:
We shouldn't reformat/reorder the entire document at the same time we make changes. It's makes the diff really confusing. If clients want to pull the changes into custom templates it will be a pain.

I'd personally _only_ add the new fields for the language files.
If we want to reorganize the fields to be alphabetical we should do it _as a new commit_ separate from this PR.